src/raspi/main.py

```diff
@@ -1,177 +1,101 @@
 from time import sleep
-from control.muscle.muscle_controller import contract, relax
-from io_controller.pneumatics.compressor import (
-    turn_compressor_off,
-    turn_compressor_on,
-)
-from control.motor.motor_controller import MotorController
-from utils.cpu import setup_cpu
-from utils.interval import clear_intervals
-from utils.util import *
-from io_controller.pneumatics.pressure import (
-    is_pressure_ok,
-    on_pressure_active,
-    on_pressure_deactive,
-)
-import pygame
 
-SPEED = 0.1
+from component import MotorController, compressor_actions, muscle_actions
+from component.muscle.pneumatics import pressure_actions
+from state_management import (
+    clear_intervals,
+    configure_device,
+    configure_logger,
+    setup_cpu,
+)
+from view.pygame import *
```

Comment:
Keeping pygame might not need to happen if we run a non-GUI OS on the Raspi

Reply:
Resolved by default due to Irrelevant conversation to this PR.